src/Core/Application/GlobalUsings.cs

```diff
@@ -14,4 +14,5 @@
 global using FSH.WebApi.Shared.Notifications;
 global using MediatR;
 global using Microsoft.Extensions.Localization;
-global using Microsoft.Extensions.Logging;
\ No newline at end of file
+global using Microsoft.Extensions.Logging;
+global using Microsoft.Extensions.DependencyInjection;
```

Comment:
Where is this used in application? It shouldn't be that many places... don't think it warrants a global using...